src/renderers/WebGLRenderer.js

```diff
@@ -57,6 +57,58 @@ import { WebGLUniformsGroups } from './webgl/WebGLUniformsGroups.js';
 import { createCanvasElement, probeAsync, warnOnce } from '../utils.js';
 import { ColorManagement } from '../math/ColorManagement.js';
 
+function toReversedProjectionMatrix( projectionMatrix ) {
```

Comment:
Can we put these functions into `src/utils.js`?

Reply:
I can split the [-1, 1] -> [0, 1] and [0, 1] -> [1, 0] conversions into their own methods depending on https://github.com/mrdoob/three.js/pull/29445#discussion_r1767708597.

Is that a good place if we want to use this for WebGPU code later?

Reply:
Yes, `utils.js` is already used in `WebGPURenderer`.